src/index.js

```diff
@@ -32,7 +32,7 @@ function makeOptions (userOptions) {
   }
 
   // Copy over non-standard options.
-  for (let name in userOptions) {
+  for (let name of Object.getOwnPropertyNames(userOptions)) {
```

Comment:
You might be able to `assign(options, userOptions)` (via `import assign from 'object-assign') here instead which may fix the CI (might cause other issues we don't know about).